src/Services/FeedbackService.php

```diff
@@ -14,6 +14,8 @@
 use Plenty\Modules\Item\Attribute\Contracts\AttributeValueNameRepositoryContract;
 use Plenty\Modules\Item\Item\Contracts\ItemRepositoryContract;
 use Plenty\Modules\Item\Variation\Contracts\VariationRepositoryContract;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
```

Comment:
Is this usable in plugins?

Reply:
Right now yes, in the "feedbackExists" method.

Reply:
I meant if the ModelNotFoundException is in the plugin interface to be used inside plugins